pkg/webui/console/containers/api-key-modal-form/create.js

```diff
@@ -0,0 +1,149 @@
+// Copyright © 2025 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React, { useCallback } from 'react'
+import { defineMessages } from 'react-intl'
+
+import tts from '@console/api/tts'
+import { APPLICATION, GATEWAY, ORGANIZATION, USER } from '@console/constants/entities'
+
+import FormField from '@ttn-lw/components/form/field'
+import FormSubmit from '@ttn-lw/components/form/submit'
+import SubmitButton from '@ttn-lw/components/submit-button'
+import Input from '@ttn-lw/components/input'
+import Button from '@ttn-lw/components/button'
+import { useFormContext } from '@ttn-lw/components/form'
+import RightsGroupModal from '@ttn-lw/components/rights-group-modal'
+import toast from '@ttn-lw/components/toast'
+
+import sharedMessages from '@ttn-lw/lib/shared-messages'
+import PropTypes from '@ttn-lw/lib/prop-types'
+
+import ApiKeyForm from './form'
+import validationSchema from './validation-schema'
+import { encodeExpiryDate, decodeExpiryDate } from './utils'
+
+const sdkServices = {
+  [APPLICATION]: 'Applications',
+  [GATEWAY]: 'Gateways',
+  [ORGANIZATION]: 'Organizations',
+  [USER]: 'Users',
+}
```

Comment:
Can you double check if we don't have these constants defines elsewhere? If it's already in global constants use than one, if we have duplicates converge that into the global constants, otherwise, let's actually add this to the constants.